sources/osg/BufferArray.js

```diff
@@ -124,11 +130,17 @@ MACROUTILS.createPrototypeObject(
             return false;
         },
 
+        invalidate: function() {
```

Comment:
invalidate sounds good to me. another option could be "clear", as it seems to discard the underlying data.